doc/bucket-class-crd.md

```diff
@@ -1,73 +1,75 @@
+- Negative (e.g. `-1`) - when the user knows there are no out of band writes, they can use a negative TTL, which means no revalidations are done; if the object is in the cache - it is returned without an ETag comparison. This is the most performant option.
+- Zero (`0`) - the cache will always compare the object's ETag before returning it. This option has a performance cost of getting the ETag from the remote target on each object read. This is the least performant option.
+- Positive (denoted in milliseconds, e.g. `3600000` equals to an hour) - once an object was read once and saved in the cache, the chosen amount of time will have to pass prior to the object's ETag being compared again.
+
+## Replication Policy
+A namespace bucket class defines a policy for namespace buckets.
+A bucket class will define a replication policy when an admin would like to replicate objects 
 within noobaa bucket to another noobaa bucket (source/destination buckets can be regular/namespace buckets).
 
 Replication policy will require the content of a JSON file which defines array of rules.
   - Each rule is an object contains rule_id, destination bucket and optional filter object that contains prefix field.
   - When a filter with prefix is defined - only objects keys that match prefix, will be replicated.
 
-Constraints:
-A backing-store name may appear in more than one bucket-class but may not appear more than once in a single bucket-class.
-The operator cli currently only supports a single tier placement-policy for a bucket-class.
-Upon creating regular buckets, the user will first need to create a placement-bucketclass which contains placemant policy.
-Upon creating namespace buckets, the user will first need to create a namespace-bucketclass which contains namespace policy.
-A namespace bucket class of type cache must contain both Placement-policy or Namespace-policy.
-A namespace bucket class of type single/multi must contain Namespace-policy.
-YAML must be used to create a bucket-class with a placement-policy that has multiple tiers.
-Placement-policy is case sensitive and should be of value (Mirror|Spread).
-Namespace-policy is case sensitive.
+## Constraints:
+- A backing store name may appear in more than one bucket class but may not appear more than once in a single bucket class.
+- The operator CLI currently only supports a single tier placement-policy for a bucket class.
+- Upon creating regular buckets, the user will first need to create a placement bucketclass which contains a placemant policy.
+- Upon creating namespace buckets, the user will first need to create a namespace bucketclass which contains namespace policy.
+- A namespace bucket class of type cache must contain both a placement and a namespace policy.
+- A namespace bucket class of type single/multi must contain Namespace-policy.
+- YAML must be used to create a bucket class with a placement-policy that has multiple tiers.
```

Comment:
Maybe mention the YAML also in the previous section? (not only in the constraints section)